xcp_d/interfaces/prepostcleaning.py

```diff
@@ -56,38 +69,36 @@ class _RemoveTROutputSpec(TraitedSpec):
         usedefault=True,
     )
 
+    dummy_scans = traits.Int(desc="Number of volumes dropped.")
```

Comment:
Not entirely sure what this input spec would produce differently for auto dummyscans/ if someone provided dummytime?

Reply:
At the moment, this assumes that dummytime (if provided) was converted into dummy scans in the nifti/cifti postprocessing workflow. Really, the only thing that this step does to dummy scans is it loads it from auto if necessary.